### ISSUE TITLE
Preventing Close when Submitting a form

### ISSUE THREAD

**Issue by jelewis8** [cbopp-art/lightcase#27]
It seems that when you have a form inside a lightcase instance and you submit the form, that triggers the close() on the whole lightcase.

Not sure if that's a bug or not, but it's probably not optimal.

**Comment by cbopp-art** [comment]
@jelewis8 
Are you sending the form using AJAX?
Have a look how it is done at the form on demo page: http://cornel.bopp-art.com/lightcase/

**Comment by jelewis8** [comment]
Yes, Ajax. It's not referencing a form directly--rather the form is inside
an inline div that gets loaded in lightcase.

It's a CF7 form if that makes a difference.

**Comment by cbopp-art** [comment]
Could you share a link to your demo? Otherwise it is quite hard to reproduce the issue you got.
Thanks!

**Comment by jelewis8** [comment]
Sure.
LINK: http://www.beverlyhillscc.com/

Click on the SUBMIT INFORMATION ONLINE link to bring up the lightbox.

I've left the relevant javascript in the footer.

**Comment by cbopp-art** [comment]
Strange, but I can't send your form, nothing happens if I click the submit button. I tested on OSX Safari and Chrome.
Are you sure that your form is working properly?

**Comment by jelewis8** [comment]
If you submit, the lightbox disappears. If you reopen you can see the form
has gone through validation (I have required fields) so I know the form
itself is working.

But the lightbox itself isn't persistent.

**Comment by cbopp-art** [comment]
Actually, I can't really test your form. On OSX, the form cannot be sent. There are just console logs and some Javascript warns and errors from e.g. jquery-migrate-1.2.1.js, jquery.min.js and jquery.gtip.min.map

**Comment by jelewis8** [comment]
I think I worked it out. I just set up a direct form page and linked to
that, and that seems to have addressed the issue.

Thanks!

**Comment by cbopp-art** [comment]
Alright! so, therefore I'm going to close this issue.

**Comment by tatimblin** [comment]
I was stuck on this same problem, and wanted to share what went wrong for me, since @jelewis8 ended up doing something different. 
I needed a form with error feedback while in Lightcase, but the ajax enabled form would fail to submit anything. Turns out all the Lightcase content was rendered with `display: none;` so Lightcase could change it to `display: block;` when opened, but this prevented the ajax request from going through. To solve this I removed the `display: none;` and hid the content by controlling `height` and `overflow`, but anyway other than `display: none;` would work.
